configs/cost-management.json

```diff
@@ -3,31 +3,31 @@
         "name": "Cost Administrator",
         "description": "A cost management administrator role that grants read and write permissions.",
         "system": true,
-        "version": 2,
+        "version": 3,
```

Comment:
I don't think cost needs a version bump since nothing is changing in here, right?